### PR TITLE
feat: ドキュメントサイト

### LINKED ISSUE #313: コマンド・機能ドキュメント
mkdocsか何かを使って、コマンドのドキュメントを書けるようにしたい。

### PR DESCRIPTION
**Pull request by book000**
- close #313 

## コマンド

- [x] alpha
- [x] bassline
- [x] birthday
- [x] controrandja
- [x] origin
- [x] ping
- [x] potato
- [x] powa
- [x] super
- [x] tmttmt
- [x] toarja
- [x] toar
- [x] tochaos
- [x] toheja
- [x] tohe
- [x] tojaen
- [x] toja
- [x] tokoja
- [x] torandja
- [x] toswja
- [x] tozhja
- [x] tozh
- [x] translate

## 機能

- [x] everyday
- [x] greeting
- [x] joined-notifier
- [x] leaved-notifier
- [x] meeting-vote
- [x] new-discussion-mention
- [x] pin-prefix
- [x] pin-reaction
- [x] vc-speech-log-url

### CODE REVIEW

**Comment by book000** [comment]
`// TODO` の内容を確認。

File: docs/commands/bassline.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: bassline
+template: command.html
+---
+
+ベースラインパーティー中の臨時ニュースを投稿します。
+
+引数に応じて、投稿される内容が変わります。
```
Comment (book000) [inline docs/commands/bassline.md:8]: この一文いらないな

File: docs/commands/alpha.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: alpha
+template: command.html
+---
+
+このコマンドを実行すると、以下のメッセージをチャンネルに投稿します。
+
+```text
```
Comment (book000) [inline docs/commands/alpha.md:8]: 使い方セクションに持ってってもよさそう

File: docs/commands/powa.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+`/powa` と実行すると、`ポわ～～～～～～～ｗｗｗｗ！！！ｗ！ｗｗ！ｗ！ｗ` を投稿します。
+
+## 必要な権限
+
+誰でも利用できます。
+
+## 関連情報
+
+- [ソースコード](https://github.com/jaoafa/jaotan.ts/blob/master/src/commands/powa.ts)
+
```
Comment (book000) [inline docs/commands/powa.md:19]: 無駄な改行？

File: docs/commands/translate.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+title: translate
+template: command.html
+---
+
+Google Translate サービスを利用して、翻訳をします。
+
+## 使い方
+
+このコマンドは、`/translate <翻訳元の言語> <翻訳後の言語> <テキスト>` と実行します。  
+`<` `>` の箇所は以下の参考に入力します。
```
Comment (book000) [inline docs/commands/translate.md:11]:
```suggestion
`<` `>` の箇所は以下を参考に入力します。
```

**Comment by book000** [comment]
@jaotan review

**Review by jaotan — APPROVED** [review]
It was reviewed by a request by comment.

**Pull request merged**